builder/handlebars/index.hbs

```diff
@@ -111,6 +111,9 @@
       {{#if example}}
         <div class="kss-modifier__wrapper">
           <div class="kss-modifier__heading kss-style">
+              <a href="#kssref-{{referenceURI}}-fullscreen"
+                 onclick="var el = document.getElementsByClassName('kss-{{referenceURI}}-fullscreen')[0]; screenfull.toggle(el); return false;"
+                      >Focus</a>
```

Comment:
I don't see how this will work. If the section has multiple modifiers, getElementsByClassName will return an array of dom elements.